lib/super_digit.rb

```diff
@@ -1,15 +1,29 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n) where n is the number of digits (Is there a way to be more specific?)
+# Space Complexity - O(n) where n is the number of digits; using call stack + auxiliary array
 def super_digit(n)
```

Comment:
Note that this works, and the method has an O(n) loop and that's nested inside a recursive call.  So your time/space complexities are off.